data/json/items/melee.json

```diff
@@ -3349,7 +3349,57 @@
         "to_hit": 2,
         "techniques" : ["PRECISE", "RAPID", "WBLOCK_2"],
         "flags": ["STAB", "SHEATH_SWORD"]
-    },
+    },{
+    "id": "broadsword_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "broadsword",
+    "description": "This is an early modern sword seeing use in the 16th, 17th, and 18th centuries.  This sword appears to be made very poorly, but it should still stand up to a few swings.",
+    "price": 120000,
+    "material": "budget_steel",
+    "flags" : ["CHOP", "SHEATH_SWORD", "IAIJUTSU"],
+    "techniques": "WBLOCK_1",
+    "weight": 1133,
+    "volume": 7,
+    "bashing": 7,
+    "cutting": 29,
+    "to_hit": 2,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "use_action": { "type": "knife" },
+    "category" : "weapons",
+    "qualities": [["CUT", 1], ["BUTCHER", 9]]
+  },{
+    "id": "macuahuitl",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "Macuahuitl",
+    "description": "This is an ancient sword that was once used by the aztecs. Its edge menaces with shiny flakes of obsidian. Even the gentlest of touches still breaks your skin.",
+    "price": 800000,
+    "material": "obsidian, wood",
+    "flags" : ["CHOP", "SHEATH_SWORD"],
+    "techniques": "WBLOCK_1",
+    "weight": 1133,
+    "volume": 7,
+    "bashing": 8,
+    "cutting": 45,
```

Comment:
Obsidian isn't really a monomolecular high-tech material, it is just volcanic glass that can be sharpened really well.